### PR TITLE
Upgraded flink to 1.17.1

### PR DESCRIPTION
**Pull request by isterin**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -69,36 +69,36 @@ val flinkDeps =
...
-    "org.apache.flink"  % "flink-connector-aws-kinesis-firehose" % V.flink % Provided,
-    "org.apache.flink" %% "flink-connector-cassandra"            % V.flink % Provided,
-    "org.apache.flink"  % "flink-connector-elasticsearch7"       % V.flink % Provided,
-    "org.apache.flink"  % "flink-connector-jdbc"                 % V.flink % Provided,
-    "org.apache.flink"  % "flink-connector-rabbitmq"             % V.flink % Provided,
+    "org.apache.flink"  % "flink-connector-base"                 % V.flink      % Provided, // ds hybrid source
+    "org.apache.flink"  % "flink-connector-files"                % V.flink      % Provided, // ds text files
+    "org.apache.flink"  % "flink-parquet"                        % V.flink      % Provided, // parquet bulk sink
+    "org.apache.flink"  % "flink-connector-kafka"                % V.flink      % Provided,
+    "org.apache.flink"  % "flink-connector-kinesis"              % "4.1.0-1.17" % Provided,
```
Comment (nextdude) [inline build.sbt:80]: Can we replace the `1.17` here with `flinkMinor`?

**Review by nextdude — CHANGES_REQUESTED** [review]
Except for one minor change, looks good.

File: build.sbt (hunk 1 above)
Comment (isterin) [reply build.sbt:80]: Yes

Comment (isterin) [reply build.sbt:80]: Doing now

**Pull request merged**